data/cameras.xsd

```diff
@@ -292,7 +292,7 @@
   </xs:simpleType>
   <xs:simpleType name="AliasTypeIdType">
     <xs:restriction base="xs:string">
-      <xs:pattern value="[a-zA-Z0-9 \-]{7,21}"/>
+      <xs:pattern value="[a-zA-Z0-9 \-/]{7,21}"/>
```

Comment:
No, i really mean `\/`.
`/ An unescaped delimiter must be escaped; in most languages with a backslash (\)`

Reply:
It's not escaped in any other model expressions AFAICT?

I've done it for the Alias id attribute instead of the actual model string...

Reply:
> It's not escaped in any other model expressions AFAICT?

Oh how fun, https://regex101.com/ complained without an escape...